README.md

```diff
@@ -8,3 +8,4 @@ A static site for practicing GitHub collaboration
 * Push your branch to the remote: `$ git push origin [branch name]`
 * Create a pull request on the GitHub UI
 * Wait for insightful comments from your teammates :)
+* Celebrate with your teammates 
```

Comment:
Let's add some detail for the type of celebration. Then again sometimes I've been accused of being a party pooper

Reply:
Love that Miles!

Reply:
Great idea -- did you have something in mind?